### ISSUE TITLE
 keyboard shortcuts on gif or picture or video

### ISSUE THREAD

**Issue by TimAimee** [philc/vimium#2176]
can you add the keyboard shortcuts on gif or picture or video?
just like that!
[http://ww1.sinaimg.cn/mw1024/a1ca3812gw1f56s3kd8dbj20fa0byq4z.jpg](url)

**Comment by Antinatalist** [comment]
yep, I also don't know how to play/stop/"escape" from embeded videos/music files.

**Comment by philc** [comment]
Closing due to age, and also unfortunately I can't tell what the original issue is now given that the sites at the URLs have changed.
